### PR TITLE
task/GetAccessButtonTest_and_SynapseLoginTest

### PR DESCRIPTION
**Pull request by haraprasadj**
Added executable tests for Get Access button and Synapse Login username. 
Please review @mfshao @frickjack @atharvar28 @themarcelor

### CODE REVIEW

File: suites/portal/fileExplorerTest.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+/*
+Data Setup:
+    1. Commons_1 - URL configured
+        i) User_11 - No access to some data
+        ii) User_12 - Access to all data
+    2. Commons_2 - URL not configured
+        i) User_21 - No access to some data
+        ii) User_22 - Access to all data
+ */
```
Comment (atharvar28) [inline suites/portal/fileExplorerTest.js]: Not a blocker, but is there a way to make this less complex?

File: suites/portal/homepageTest.js [hunk 2]
```diff
@@ -6,3 +6,17 @@ Scenario('login @portal', (home) => {
   home.ask.seeDetails();
   home.complete.logout();
 });
+
+/*
+Data Setup:
+    1. User_1 - Synapse account having access to a project in a brain commons (Commons_1)
```
Comment (atharvar28) [inline suites/portal/homepageTest.js]: Similarly here, can you add the user details inside the Scenarios as a part of steps?

File: suites/portal/fileExplorerTest.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+Feature('File Explorer');
```
Comment (themarcelor) [inline suites/portal/fileExplorerTest.js]: Looks like other test scripts have a reference to the corresponding JIRA ID that tracks this task, maybe we should follow this standard:
e.g.,
> "Feature('<some description> - https://ctds-planx.atlassian.net/browse/<JIRA ID>');"

File: suites/portal/fileExplorerTest.js (hunk 1 above)
Comment (themarcelor) [reply suites/portal/fileExplorerTest.js]: @haraprasadj , perhaps you could add a sample URL.
Take, per instance, the Guppy-related tests that have been merged recently:
https://github.com/uc-cdis/gen3-qa/pull/183/files
It has lines like:
> "+        Take qa-mickey as example:"
> "+        1. Goto target commons' query page (e.g., https://qa-mickey.planx-pla.net/query)."

Comment (haraprasadj) [reply suites/portal/fileExplorerTest.js]: @atharvar28 will describe the data a bit more in the scenario (added as a separate section to help in automation later)
@themarcelor will do that

File: suites/portal/homepageTest.js (hunk 2 above)
Comment (haraprasadj) [reply suites/portal/homepageTest.js]: @atharvar28 will do

File: suites/portal/fileExplorerTest.js (hunk 3 above)
Comment (haraprasadj) [reply suites/portal/fileExplorerTest.js]: @themarcelor, will do

**Pull request merged**